.github/workflows/helm_charts_scalar.yml

```diff
@@ -170,10 +172,11 @@ jobs:
           fi
 
       - name: Create kind ${{ matrix.k8s }} cluster
-        uses: helm/kind-action@v1.4.0
+        uses: helm/kind-action@v1.7.0
         with:
           config: .github/kind-cluster.yaml
           node_image: kindest/node:${{ matrix.k8s }}
+          kubectl_version: ${{ matrix.k8s }}
```

Comment:
I specify the `kubectl` command version explicitly to use the same version of the Kubernetes cluster.